flake8_spellcheck/__init__.py

```diff
@@ -129,17 +129,21 @@ def _detect_errors(self, tokens, use_symbols, token_type):
         code = get_code(token_type)
 
         for pos, token in tokens:
+            value = token
             if use_symbols:
-                valid = token.lower() in self.words
+                if token_type == tokenize.COMMENT:
+                    # strip the word of surrounding quotes
+                    value = value.strip("'")
```

Comment:
I'm not understanding why you don't also apply the same behaviour to `"` and ` here

Reply:
I thought of doing that but they seem to be handled correctly already (see the corresponding test case - I have examples of `"` and ` there

Reply:
That doesn't seem right - could you try adding a few more test cases to confirm this? I'm not seeing anything in the code that handles these cases.